api/queries_repo.go

```diff
@@ -845,100 +849,75 @@ type RepoMetadataInput struct {
 
 // RepoMetadata pre-fetches the metadata for attaching to issues and pull requests
 func RepoMetadata(client *Client, repo ghrepo.Interface, input RepoMetadataInput) (*RepoMetadataResult, error) {
-	result := RepoMetadataResult{}
-	errc := make(chan error)
-	count := 0
+	var result RepoMetadataResult
+
+	g, _ := errgroup.WithContext(context.Background())
```

Comment:
`errgroup` felt like a nice improvement here.

Reply:
Agreed. Nit: since we don't have a parent context here, and we won't use the child context for anything either, you can just initialize a Group without context.
```suggestion
	var g errgroup.Group
```